src/bokeh/plotting/_geo_feature.py

```
@@ -0,0 +1,538 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# External imports
+import numpy as np
+
+try:
+    import cartopy.crs as ccrs
+    import cartopy.feature as cfeature
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "The module 'bokeh.plotting._geo_feature' requires the optional dependency 'cartopy'."
+        "To use this module, please install 'cartopy'.",
+    ) from e
+
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+GEO_FEATURE_COLORS = {
+    "LAND": "#EFEFDB",
+    "STATES": "#EFEFDB",
+    "RIVERS": "#9FDBF3",
+    "OCEAN": "#9FDBF3",
+    "LAKES": "#9FDBF3",
+    "BORDERS": "#CCCCCC",
+    "MAP_BOUNDARYS": "#000000",
+    "COASTLINES": "#000000",
+}
```

@bryevdv Sorry for not being clear. I just wanted to ask if the definition of the default colors here is acceptable. Maybe a better solution exist. In an earlier comment you mentioned a possibility to use a theme. The current solution gives a user the chance to overwrite the defaults and use his favorite colors for all geographical plots.

---

Are we viewing these as user-configurable global defaults? I'd say a dataclass would be nicer here, then there is no chance of users typing the field name and having it just silently fail.

---

FWIW to get theming I think we'd have to make this into a Bokeh model, and defer usage of the values until the BokehJS side of things. We'd also have to make sure its shipped with the document to begin with, e.g. maybe `Figure` would have to get an `Instance(GeoColors)` property, etc. 

I'll leave it to you to decide which path to take, otherwise LGTM